### DISCUSSION TITLE
Exam Subjects

### DISCUSSION THREAD

**Discussion by cuhlig42** [JCluzet/42_EXAM#52] [Q&A] [answered]
Hello now that the program is running I wanted to ask. 
Does  the Piscine Part Week 01 and 02 inlcludes all this 4 exams of your github site?
![error 7](https://user-images.githubusercontent.com/113346209/212740116-673ac9d5-3615-434c-a83a-5307427135db.png)

**Comment by JCluzet** [comment, marked as answer]
Hi @cuhlig42 👋
Before there were the 4 pool exam weekends on 42_EXAM.
However I removed the possibility to do the 3 and 4 exam, in order to give an equal chance to the candidates. 
Indeed I left the directories in the Github. 
Before there was a mention why the week 3 and 4 were not available, but it has been removed in V2.

So no, Week1 is the first file.
And Week2 is the second.
Folder 3 and 4 are not used, maybe it can change in the future, but to let a chance to 42_EXAM to survive and not to be deleted by 42, I preferred to remove them.

**Reply by cuhlig42** [reply]
ahh alright ty
